GPL/DemanglerGnu/src/demangler_gnu_v2_33_1/c/missing.c

```diff
@@ -68,7 +68,7 @@ xrealloc (ptr, size)
   
```

Comment:
Seems like a better fix would be to declare ptr as a void* parameter.  It's missing for some reason.